export/wiki/species/xp.py

```diff
@@ -19,29 +21,34 @@ class LevelExperienceRampType(Enum):
     MAX = 4
 
 
+OFFICIAL_SERVER_MAX_TAME_LEVEL = 450
+
+
 class LevelData(ExportModel):
-    ramp: str = Field(
+    wildOverride: Optional[IntLike] = None
+    wildLevelTable: List[MinMaxChanceRange] = [
+        MinMaxChanceRange(chance=0.5405405, min=1.0, max=5.0),  # weight = 1.0
+        MinMaxChanceRange(chance=0.2702703, min=6.0, max=12.0),  # weight = 0.5
+        MinMaxChanceRange(chance=0.1351351, min=13.0, max=20.0),  # weight = 0.25
+        MinMaxChanceRange(chance=0.05405405, min=21.0, max=30.0),  # weight = 0.1
+    ]
+
+    experienceRamp: str = Field(
         'DinoEasy',
         description="Name of ramp that describes amount of experience needed to progress.",
     )
-    maxExperience: Optional[FloatProperty]
-    maxLevels: Optional[int] = Field(
-        None,
-        title="Max Level Ups",
+    maxExperience: FloatLike = 3550010
+    maxTameLevels: int = Field(
+        73,
         description="Max amount of level ups this species can have at default server settings.",
     )
-    capOffset: Optional[IntProperty] = Field(
-        None,
-        title="Tame Level Cap Offset",
-        description=
-        "Number of extra levels (above normal cap) this species can gain when tamed without getting destroyed by validation",
+    officialCap: IntLike = Field(
```

Comment:
Althought this might be easier to read for a human I think the idea of hardcoding the 450 level cap inside the extractor is a bad idea in the long term.